src/MySqlConnector/Core/SchemaProvider.cs

```diff
@@ -106,7 +145,7 @@ private Task FillDataSourceInformation(IOBehavior ioBehavior, DataTable dataTabl
 		static string GetVersion(Version v) => "{0:00}.{1:00}.{2:0000}".FormatInvariant(v.Major, v.Minor, v.Build);
 	}
 
-	private Task FillMetadataCollections(IOBehavior ioBehavior, DataTable dataTable, CancellationToken cancellationToken)
+	private Task FillMetadataCollections(IOBehavior ioBehavior, DataTable dataTable, string?[] restrictionValues, CancellationToken cancellationToken)
```

Comment:
This method need to be updated to return the `NumberOfRestrictions` for each schema collection; it's currently hard-coded to 0.

Reply:
i changed it